README.md

```diff
@@ -210,6 +210,95 @@ window.geofence.getWatched().then(function (geofencesJson) {
 });
 ```
 
+## Frequency Limiting
+
+```javascript
+notification: {
+    frequency: Number(600), //Seconds required between retriggering a fence.
+}
+```
+
+## Using On/Off Schedule Data
+
+ScheduleData is an array consisting of a start and end time for each day. Array begins on Sunday. All times are local to the device.
+
+```javascript
+notification: {
+    scheduleData: [
```

Comment:
I'm thinking how to make this simpler. How about:

```
notification: {
     weekSchedule: {
           monday: "8:00-16:30",
           tuesday: "9:00-14:00",
           .....
     }
```

but actually, parse this in javascript and send to a Java code using your way.